src/main/java/de/zalando/tip/zalenium/servlet/live.java

```diff
@@ -0,0 +1,123 @@
+package de.zalando.tip.zalenium.servlet;
+
+/*
+    This class renders an HTML with a similar appearance to the Grid Console, it just adds an iFrame that
+    allows users to see what is happening inside the container while they run their tests.
+    The code here is based on the ConsoleServlet class from the Selenium Grid
+ */
+
+import com.google.common.io.ByteStreams;
+import de.zalando.tip.zalenium.proxy.DockerSeleniumRemoteProxy;
+import org.openqa.grid.internal.Registry;
+import org.openqa.grid.internal.RemoteProxy;
+import org.openqa.grid.internal.utils.HtmlRenderer;
+import org.openqa.grid.web.servlet.RegistryBasedServlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+// We use this class name to be able to go to the resource like this: http://localhost:4444/grid/admin/live
+public class live extends RegistryBasedServlet {
+
+    public live(){
+        this(null);
+    }
+
+    public live(Registry registry) {
+        super(registry);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        process(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        process(request, response);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a "try/catch" block for "process". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1989)